Content.Server/GameObjects/Components/Power/BaseNetConnectorComponent.cs

```diff
@@ -104,7 +104,7 @@ private void SetVoltage(Voltage newVoltage)
         }
     }
 
-    public enum Voltage
+    public enum Voltage : byte
```

Comment:
I assume these values are in bounds...

Reply:
Yes. This enum here has three values: High, Meduium, Apc and they all in bound of ```enum``` type. All are set by NodeGroupID enum from NofeGroupFactory.cs. And this enum has 7 values that goes from 0 to 6 and is also a ```byte``` type.